diff --git a/esm/rules.js b/esm/rules.js
index 70530da07f5dc963cf0525df684fc470593cc626..b09408b7ceeb55c40f6bb142566c7de8a1371605 100644
--- a/esm/rules.js
+++ b/esm/rules.js
@@ -1,6 +1,8 @@
 import * as Yup from 'yup';
 import { isLogicRule } from './utils.js';
-import { isUndefined } from 'util';
+function isUndefined(value) {
+  return value === undefined;
+}
 export class Rule {
     constructor(name, func, constructorOptions) {
         const options = this.normalizeOptions(constructorOptions);
